README.md

```diff
@@ -1,41 +1,30 @@
-# [WIP] skupper [WIP]
+# Skupper
+Skupper enables cloud communication by enabling you to create a Virtual Application Network.
 
-Note: this repository is a work in progress targeted for the consolidation
-and refactoring of the skupper implementation.
+This upper layer network decouples addressing from the underlying network infrastructure, providing a secure and simple alternative to VPNs.
 
-Command line tool for setting up and managing skupper installations
+You can use Skupper to create simple integrations between clusters as described in the [Getting Started](https://skupper.io/start/index.html) or create a topology that supports:
```

Comment:
I don't really like the word 'integrations' here. What about something like:

You can use Skupper to create a network from namespaces on one or more Kubernetes clusters as described in .... There are no restrictions on the topology created which can include redundant paths but does not need to be a full mesh. Skupper then ensures anycast and multicast (for http) communication over that.

Or something along those lines...?

Reply:
@grs updated, lemme know if it works now?